package.json

```diff
@@ -14,64 +14,62 @@
   "dependencies": {
     "@mozilla/readability": "^0.5.0",
     "cleanslate": "^0.10.1",
-    "compute-scroll-into-view": "^3.0.0",
+    "compare-versions": "^6.1.1",
+    "compute-scroll-into-view": "^3.1.0",
     "csp-serdes": "github:cmcaine/csp-serdes",
     "css": "^3.0.0",
     "editor-adapter": "^0.0.5",
-    "esbuild": "^0.20.2",
+    "esbuild": "^0.24.0",
     "fuse.js": "^7.0.0",
     "nearley": "^2.20.1",
-    "ramda": "^0.29.1",
-    "semver-compare": "^1.0.0",
+    "qrcode": "^1.5.4",
```

Comment:
Hi :)

This was vendored on purpose to defend against qr code hijacking. It's a simple package that does one thing and does it well enough - the risk of it getting taken over without me noticing and it stealing someone's bitcoins is much scarier to me than the idea of us missing out on some tiny bug fixes